vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.kt

```diff
@@ -921,20 +931,16 @@ class VectorSettingsPreferencesFragment : PreferenceFragmentCompat(), SharedPref
                 mLoadingView = parent.findViewById(R.id.vector_settings_spinner_views)
                 parent = parent.parent as View
             }
-        }
-
-        if (null != mLoadingView) {
-            mLoadingView!!.visibility = View.VISIBLE
+        } else {
+            mLoadingView?.visibility = View.VISIBLE
```

Comment:
As a remark, the code is not equivalent as before, because in the case where `mLoadingView` it will not be set to visible.
That said, it should never happen because `mLoadingView` is retrieved in `onResume()`

Reply:
I understand. But there was case that , if it's null , then it would find `mLoadingView` from parent.

But Kotlin compiler wasn't smart enough for that . I was thinking using Kotlin Contracts

Reply:
Yes, but when found, the visibility will not be set to visible (because it's the else block now).
Anyway, it's ok to leave it like that

Reply:
When I do the work for Coroutines in CleanAsyncTask. I can do the work for Contracts . If want. Or else would just do for Coroutine .